### ISSUE TITLE
GWENT: The Witcher Card Game (1284410)

### ISSUE THREAD

**Issue by jqadev** [ValveSoftware/Proton#3890]
# Compatibility Report
- Name of the game with compatibility issues: GWENT: The Witcher Card Game
- Steam AppID of the game: 1284410

## System Information
- GPU: GeForce 930MX
- Driver/LLVM version: NVIDIA 440.64
- Kernel version: 5.4.0-29-generic
- [Link to full system information report as Gist](https://gist.github.com/jqadev/8a892c8f8b5145169b4d5ae8c142c7c9)
- Proton version: 5.0-7

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1284410.zip](https://github.com/ValveSoftware/Proton/files/4651180/steam-1284410.zip)

## Symptoms 
The game doesn't launch.
I guess that the reason is explained by this log: 
```
21919.811:0118:011c:err:seh:setup_exception stack overflow 1632 bytes in thread 011c eip 00007f45f81b3228 esp 0000000000130fb0 stack 0x130000-0x131000-0x230000
```

**Comment by Leopard1907** [comment]
https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/5.8-GE-2-MF

Game fully works with this release , at least on my quick testing. Gameplay works , intro videos etc works. Which is known that this game needs mfp.

**Comment by jqadev** [comment]
> https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/5.8-GE-2-MF
> 
> Game fully works with this release , at least on my quick testing. Gameplay works , intro videos etc works. Which is known that this game needs mfp.

It also doesn't launch for me with 5.8-GE-2-MF but it resolves another issue reported by me, i.e. https://github.com/ValveSoftware/Proton/issues/350#issuecomment-630328647 :) Thank you! :+1:

**Comment by thaewrapt** [comment]
Works without any workarounds on 5.6.13-ck, Arch Linux, NVidia Optimus 440.82-1, Xorg 1.20.8-2, Proton default (5.0-7 at the moment). 

The only issue (**UPD**: happened only once on the very first run, second one went smoothly) is a non-clean exit as Steam believes the app is still open after an exit from the game and the disappearance of the game's window.

**UPD2**: It seems like no intro videos for me due to in-game progress, so can't test this critical functionality, sadly.

**Comment by Leopard1907** [comment]
@thaewrapt Works as in , intro videos etc plays or not? I think they shouldn't with stock Proton

**Comment by thaewrapt** [comment]
@Leopard1907 I don't remember seeing any intros but I should be past tutorial already (playing on mobile client before). Is there any way to force intro showing to double-check?

**Comment by jqadev** [comment]
@Leopard1907 and @thaewrapt, could you please check if  there is something like this exception
```
21919.811:0118:011c:err:seh:setup_exception stack overflow 1632 bytes in thread 011c eip 00007f45f81b3228 esp 0000000000130fb0 stack 0x130000-0x131000-0x230000
```
 in a log file when you set `PROTON_LOG=1 %command%`?

**Comment by thaewrapt** [comment]
Updated https://github.com/ValveSoftware/Proton/issues/3890#issuecomment-630930491 as no intro videos were tested, and the on-exit issue disappeared on the second run.

**Comment by thaewrapt** [comment]
> @Leopard1907 and @thaewrapt, could you please check if there is something like this exception
> 
> ```
> 21919.811:0118:011c:err:seh:setup_exception stack overflow 1632 bytes in thread 011c eip 00007f45f81b3228 esp 0000000000130fb0 stack 0x130000-0x131000-0x230000
> ```
> 
> in a log file when you set `PROTON_LOG=1 %command%`?

Tried it, no `setup_exception` or stack overflows at all.

**Comment by tgharib** [comment]
Stuck on loading screen on linux 5.6.12-1, openSUSE Tumbleweed, nvidia 440.82, Proton 5.0-7. I've attached the log after setting `PROTON_LOG=1 %command%` if anyone is curious.

[steam-1284410.log](https://github.com/ValveSoftware/Proton/files/4652798/steam-1284410.log)

**Comment by kisak-valve** [comment]
Hello @tgharib, this looks like a line of interest from your log: `err:steamclient:create_win_interface Don't recognize interface name: SteamNetworkingSockets008`

Related: https://github.com/ValveSoftware/Proton/pull/3880

**Comment by tgharib** [comment]
@kisak-valve Hmm, I just tested with `proton-5.0-7-sdk148` and it still seems to be stuck on loading screen. I tested with `5.8-GE-2-MF` and `proton 4.11-13` and same issue in both cases.

I've attached the log with `proton-5.0-7-sdk148`.
[steam-1284410.log](https://github.com/ValveSoftware/Proton/files/4652894/steam-1284410.log)

**Comment by BillFleming** [comment]
The game seems to run fine with the 5.7 staging based releases with Guy's video patches included.
https://github.com/Frogging-Family/wine-tkg-git/releases/tag/5.7.r6.g7b0b95ee :frog:  This is an example of a prebuilt one you can use. Videos and gameplay both work fine.

**Comment by thaewrapt** [comment]
> Hello @tgharib, this looks like a line of interest from your log: `err:steamclient:create_win_interface Don't recognize interface name: SteamNetworkingSockets008`
> 
> Related: #3880

This error is also present with my configuration (reported earlier as a success story w/o videos), so attaching my logfile for reference.
[steam-1284410.log](https://github.com/ValveSoftware/Proton/files/4654272/steam-1284410.log)

**Comment by ghost** [comment]
@jqadev Have you tried deleting the prefix created by Proton and let Proton GE create the prefix? I remember reading somewhere (probably reddit), that doing so fixed their problem.

**Comment by jqadev** [comment]
@fureloka you are a master :D I've removed directory `1284410` from 
`/.steam/debian-installation/steamapps/compatdata` and run the game again with Proton GE and it works now! Thank you! :+1:

**Comment by wheatdog** [comment]
Hi all, I have tested with `proton-5.0-7-sdk148`, and `5.8-GE-2-MF`, but online mode works in none of them. It always goes back to the main menu when it finds an opponent. I have tried seasonal, classic mode, and even challenge with a friend. All of them suffer from the same issue. On the other hand, "Training" mode works flawlessly. Not sure why only the online multiplayer part isn't functional. I can't find other useful information in log. Any help? Thanks!

[steam-1284410.log](https://github.com/ValveSoftware/Proton/files/4662768/steam-1284410.log)

Edit: I also discussed this issue on [/r/gwent](https://www.reddit.com/r/gwent/comments/gmnl88/psa_gwent_now_runs_perfectly_on_linux/fr8t8c4?utm_source=share&utm_medium=web2x) and [/r/linux_gaming](https://www.reddit.com/r/linux_gaming/comments/gmmxj2/gwent_the_witcher_card_game_runing_in_linux_with/fr929r7?utm_source=share&utm_medium=web2x)

**Comment by Leopard1907** [comment]
Online portion is functional. I played like 30 matches as of now.

Nuke the prefix and start the game with GloriousEgroll's 5.8 build. It works for me since day 1 with this mentioned build.

**Comment by joelfcoelho** [comment]
Also stuck in the loading screen. No option to log in shows up.
Tried with both 5.0.7 and 5.8, as well as removing the proton prefix.

**Comment by wheatdog** [comment]
I tried remove the prefix and rerun again, but the issue remains.

Edit: Sometimes, it showed a window in game with title "unknown error" and content "there was an error connecting with one of our services. But fear not - chances are we're already fixing it".

**Comment by BillFleming** [comment]
I finally tested online play with the proton_tkg_5.7.r6.g7b0b95ee.release I was using and it works just fine. Now just gotta relearn the game better since many of the gameplay mechanics changed when it went out of beta....

**Comment by tgharib** [comment]
> Also stuck in the loading screen. No option to log in shows up.
> Tried with both 5.0.7 and 5.8, as well as removing the proton prefix.

What is your driver version, kernel version and distro? I'm curious if we have anything in common.

**Comment by joelfcoelho** [comment]
> > Also stuck in the loading screen. No option to log in shows up.
> > Tried with both 5.0.7 and 5.8, as well as removing the proton prefix.
> 
> What is your driver version, kernel version and distro? I'm curious if we have anything in common.

Arch, on 5.6.13-arch1-1 with mesa 20.0.7

**Comment by wheatdog** [comment]
I still can't play online mode. It just failed silently. I suspect that might be caused by the version of gnutls. Following is mine. Can anyone that has a working online play experience share theirs?
```
$ ps aux | grep GWENT
wheatdog 2243740  0.3  0.2  42416 16192 ?        S    23:49   0:00 python3 /home/wheatdog/.local/share/Steam/compatibilitytools.d/proton_tkg_5.8.r0.g7d92cd31/proton waitforexitandrun /home/wheatdog/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/Gwent.exe
wheatdog 2243765  1.3  0.3 1827732 23952 ?       Sl   23:49   0:00 steam /home/wheatdog/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/Gwent.exe
wheatdog 2243840 85.8  9.5 5200056 757432 ?      Sl   23:49   0:15 Z:\home\wheatdog\.local\share\Steam\steamapps\common\GWENT The Witcher Card Game\Gwent.exe
wheatdog 2244118  0.0  0.0   6636  2436 pts/1    S+   23:49   0:00 grep GWENT
$ lsof -p 2243840 | grep gnutls
Gwent.exe 2243840 wheatdog  mem       REG              259,2    1840376  9065654 /usr/lib/libgnutls.so.30.27.0
```

**Comment by W00glin** [comment]
I hate to bump an older thread but I am hoping that someone could give me a hand with troubleshooting. I installed Gwent and could not get it to load. I renamed the old intro videos in the steamapps directory and finally got it to load. I make it partially through the tutorial until it comes to the portion where I am supposed to hit the "pass" button. I click it but it does not register. And nothing I can do seems to work.

I would also like to add that the cards themselves do not have any of the photos/images on them. I can play them and interact with them but no images are displayed. Just the cards backgrounds. I cannot see the characters in the corners of the screen either.

I am running on Pop!_OS 20.04 and I have tried Proton 5.0-8 and Proton-GE 5.9 and 5.9 I am also using Intel Integrated graphics. I should mentioned I have only tried this on my laptop and not my desktop with a dedicated GPU.

Any ideas on tips I should try?

Here is a link to my gist: https://gist.github.com/W00glin/e5174e005d8a293a8ee8aefb5a013c28

Thanks @kisak-valve!

**Comment by kisak-valve** [comment]
Hello @W00glin, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by tgharib** [comment]
> I still can't play online mode. It just failed silently. I suspect that might be caused by the version of gnutls. Following is mine. Can anyone that has a working online play experience share theirs?
> 
> ```
> $ ps aux | grep GWENT
> wheatdog 2243740  0.3  0.2  42416 16192 ?        S    23:49   0:00 python3 /home/wheatdog/.local/share/Steam/compatibilitytools.d/proton_tkg_5.8.r0.g7d92cd31/proton waitforexitandrun /home/wheatdog/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/Gwent.exe
> wheatdog 2243765  1.3  0.3 1827732 23952 ?       Sl   23:49   0:00 steam /home/wheatdog/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/Gwent.exe
> wheatdog 2243840 85.8  9.5 5200056 757432 ?      Sl   23:49   0:15 Z:\home\wheatdog\.local\share\Steam\steamapps\common\GWENT The Witcher Card Game\Gwent.exe
> wheatdog 2244118  0.0  0.0   6636  2436 pts/1    S+   23:49   0:00 grep GWENT
> $ lsof -p 2243840 | grep gnutls
> Gwent.exe 2243840 wheatdog  mem       REG              259,2    1840376  9065654 /usr/lib/libgnutls.so.30.27.0
> ```

I have the same issue. I am running Steam through Flathub's Flatpak on Fedora Silverblue 33. Setting `PROTON_LOG=1`, I find the following line in the logs: `2485.290:00b8:018c:err:winediag:SECUR32_initNTLMSP ntlm_auth was not found or is outdated. Make sure that ntlm_auth >= 3.0.25 is in your path. Usually, you can find it in the winbind package of your distribution.` I believe the issue is a missing `winbind` package.

**Comment by tgharib** [comment]
I figured out my issue. If your loading screen is just loading infinitely or you get a network connection issue before you get to the main menu, you need to make sure the steam overlay is enabled. The game opens an overlay to login into GOG and I guess I didn't see that because I had the overlay disabled which is why the game failed to connect.

By the way, I have the game working despite that `ntlm_auth` warning showing in my logs so `ntlm` is not needed for the game to work.

Now that my game is working, to answer @wheatdog's question,

```
$ lsof -p 3259 | grep gnutls
Gwent.exe 3259 owner  mem       REG              253,0   2039104  1774538 /usr/lib64/libgnutls.so.30.28.1
```

**Comment by D33M0N** [comment]
Login with GOG.COM account FAILS (404 error in the GOG platform). Using facebook and then linking it to gog.com account works, but feels like cheap way to track and identify you via facebook.

**Comment by WinterSnake** [comment]
Unable to connect to servers - can't get to the main menu.
Have tried using experimental, current, and older versions of Proton, no luck. Steam overlays are enabled as per previous comments, but that didn't help at all either.
[system information](https://gist.github.com/WinterSnake/72a7145d60e8d6ac4cc8dc012cbf4d7e) if needed.

As of now, the game is completely borked for me.

Also attaching the proton log
[steam-1284410.log](https://github.com/ValveSoftware/Proton/files/8145858/steam-1284410.log)

EDIT:
It seems that I cannot authenticate with GOG on further inspection, however I never get a pop up for GOG login. I am not sure where to go from here.

**Comment by kisak-valve** [comment]
Hello @WinterSnake, these look like some lines of interest from your log:
```
err:module:import_dll Library mfc140u.dll (which is needed by L"C:\\users\\steamuser\\AppData\\Local\\Programs\\CD Projekt Red\\REDlauncher\\REDlauncher.exe") not found
err:module:LdrInitializeThunk Importing dlls for L"C:\\users\\steamuser\\AppData\\Local\\Programs\\CD Projekt Red\\REDlauncher\\REDlauncher.exe" failed, status c0000135
```

**Comment by WinterSnake** [comment]
> Hello @WinterSnake, these look like some lines of interest from your log:
> 
> ```
> err:module:import_dll Library mfc140u.dll (which is needed by L"C:\\users\\steamuser\\AppData\\Local\\Programs\\CD Projekt Red\\REDlauncher\\REDlauncher.exe") not found
> err:module:LdrInitializeThunk Importing dlls for L"C:\\users\\steamuser\\AppData\\Local\\Programs\\CD Projekt Red\\REDlauncher\\REDlauncher.exe" failed, status c0000135
> ```

I'll have to look into it later, for now I got it working through Lutris/Galaxy Launcher

**Comment by terzag** [comment]
Hello,
Since today's update (10.5, may 10 2022), Gwent doesn't start anymore. It worked perfectly before. I've tried several Proton versions (Experimental, 7.0-2, GE) with no luck.

Using PROTON_LOG=1 doesn't generate a log file but when starting Steam from a terminal, I see that it tries to run (and apparently fails) a process [steamapps]/common/GWENT The Witcher Card Game/REDprelauncher.exe. Not sure it's always been the case or if it's new, as messages above mention a REDlauncher.exe, not RED**pre**launcher.exe.

I've seen other people mentioning the same issue on Reddit, so it seems to be widespread. I'm using Steam as a flatpak, not sure the other players with issues used the same.

<details><summary>Here is a part of what I get in the terminal, starting about from when I start the game from what I can tell (what appears before seems to be Steam initialization):</summary>

```
Installing breakpad exception handler for appid(steam)/version(1647446817)
BuildCompleteAppOverviewChange: 1025
RegisterForAppOverview 1: 114ms
RegisterForAppOverview 2: 119ms
Installing breakpad exception handler for appid(steam)/version(1647446817)
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Installing breakpad exception handler for appid(steam)/version(1647446817)
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to ProcessingInstallScript with ""
esync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to SynchronizingControllerConfig with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to SiteLicenseSeatCheckout with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to CreatingProcess with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp waiting for user response to CreatingProcess ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp continues with user response "CreatingProcess"
/bin/sh\0-c\0PROTON_LOG=1 /home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/app/share/steam/compatibilitytools.d/Proton-GE'/proton waitforexitandrun  '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'\0
Game process added : AppID 1284410 "PROTON_LOG=1 /home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/app/share/steam/compatibilitytools.d/Proton-GE'/proton waitforexitandrun  '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'", ProcID 723, IP 0.0.0.0:0
chdir /home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game
ERROR: ld.so: object '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to WaitingGameWindow with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to Completed with ""
ERROR: ld.so: object '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ProtonFixes[724] INFO: Running protonfixes
ProtonFixes[724] INFO: Running checks
ProtonFixes[724] INFO: All checks successful
ProtonFixes[724] INFO: Using global defaults for "GWENT: The Witcher Card Game" (1284410)
ProtonFixes[724] INFO: No protonfix found for "GWENT: The Witcher Card Game" (1284410)
ThreadGetProcessExitCode: no such process 727
ThreadGetProcessExitCode: no such process 725
Game process updated : AppID 1284410 "PROTON_LOG=1 /home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/app/share/steam/compatibilitytools.d/Proton-GE'/proton waitforexitandrun  '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'", ProcID 726, IP 0.0.0.0:0
ThreadGetProcessExitCode: no such process 730
ThreadGetProcessExitCode: no such process 805
ThreadGetProcessExitCode: no such process 786
ThreadGetProcessExitCode: no such process 771
ThreadGetProcessExitCode: no such process 765
ThreadGetProcessExitCode: no such process 757
ThreadGetProcessExitCode: no such process 735
ThreadGetProcessExitCode: no such process 732
ThreadGetProcessExitCode: no such process 724
Game process removed: AppID 1284410 "PROTON_LOG=1 /home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/app/share/steam/compatibilitytools.d/Proton-GE'/proton waitforexitandrun  '/home/terzag/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'", ProcID 726 
ThreadGetProcessExitCode: no such process 744
ThreadGetProcessExitCode: no such process 726
Game 1284410 created interface STEAMAPPLIST_INTERFACE_VERSION001 / AppList
Game 1284410 created interface STEAMAPPS_INTERFACE_VERSION008 / Apps
Game 1284410 created interface STEAMHTMLSURFACE_INTERFACE_VERSION_004 / HTMLSurface
Game 1284410 created interface STEAMHTTP_INTERFACE_VERSION002 / HTTP
Game 1284410 created interface STEAMINVENTORY_INTERFACE_V002 / Inventory
Game 1284410 created interface STEAMMUSICREMOTE_INTERFACE_VERSION001 / MusicRemote
Game 1284410 created interface STEAMMUSIC_INTERFACE_VERSION001 / Music
Game 1284410 created interface STEAMPARENTALSETTINGS_INTERFACE_VERSION001 / ParentalSettings
Game 1284410 created interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 / RemoteStorage
Game 1284410 created interface STEAMSCREENSHOTS_INTERFACE_VERSION003 / Screenshots
Game 1284410 created interface STEAMUGC_INTERFACE_VERSION010 / UGC
Game 1284410 created interface STEAMUSERSTATS_INTERFACE_VERSION011 / UserStats
Game 1284410 created interface STEAMVIDEO_INTERFACE_V002 / Video
Game 1284410 created interface SteamController006 / Controller
Game 1284410 created interface SteamFriends015 / Friends
Game 1284410 created interface SteamMatchMaking009 / Matchmaking
Game 1284410 created interface SteamMatchMakingServers002 / MatchmakingServers
Game 1284410 created interface SteamNetworking005 / Networking
Game 1284410 created interface SteamUser019 / User
Game 1284410 created interface SteamUtils009 / Utils
Game 1284410 method call count for IClientAppManager::GetAppInstallState : 2
Game 1284410 method call count for IClientUtils::RecordSteamInterfaceCreation : 22
Game 1284410 method call count for IClientUtils::GetSteamUILanguage : 1
Game 1284410 method call count for IClientUtils::GetAppID : 24
Game 1284410 method call count for IClientUser::GetSteamID : 1
Uploaded AppInterfaceStats to Steam
Installing breakpad exception handler for appid(steam)/version(1647446817)
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and 
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and
```

</details>

**Comment by xanscale** [comment]
<details><summary>@terzag Same problem here: up to 10.4 works perfectly, with 10.5 wont start (vanilla proton experimental)</summary>

```
Could not connect to X session manager: None of the authentication protocols specified are supported
Could not connect to X session manager: None of the authentication protocols specified are supported
Could not connect to X session manager: None of the authentication protocols specified are supported
Could not connect to X session manager: None of the authentication protocols specified are supported
Could not connect to X session manager: None of the authentication protocols specified are supported
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to ProcessingInstallScript with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to SiteLicenseSeatCheckout with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to CreatingProcess with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp waiting for user response to CreatingProcess ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp continues with user response "CreatingProcess"
/bin/sh\0-c\0/home/xan/.steam/debian-installation/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/home/xan/.steam/debian-installation/steamapps/common/SteamLinuxRuntime_soldier'/_v2-entry-point --verb=waitforexitandrun -- '/home/xan/.steam/debian-installation/steamapps/common/Proton - Experimental'/proton waitforexitandrun  '/home/xan/.steam/debian-installation/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'\0
Game process added : AppID 1284410 "/home/xan/.steam/debian-installation/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/home/xan/.steam/debian-installation/steamapps/common/SteamLinuxRuntime_soldier'/_v2-entry-point --verb=waitforexitandrun -- '/home/xan/.steam/debian-installation/steamapps/common/Proton - Experimental'/proton waitforexitandrun  '/home/xan/.steam/debian-installation/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'", ProcID 16888, IP 0.0.0.0:0
chdir /home/xan/.steam/debian-installation/steamapps/common/GWENT The Witcher Card Game
ERROR: ld.so: object '/home/xan/.steam/debian-installation/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/xan/.steam/debian-installation/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/home/xan/.steam/debian-installation/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/xan/.steam/debian-installation/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/xan/.steam/debian-installation/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
pid 16893 != 16890, skipping destruction (fork without exec?)
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to WaitingGameWindow with ""
GameAction [AppID 1284410, ActionID 1] : LaunchApp changed task to Completed with ""
esync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
ThreadGetProcessExitCode: no such process 17059
ThreadGetProcessExitCode: no such process 17057
ThreadGetProcessExitCode: no such process 17056
ThreadGetProcessExitCode: no such process 17055
ThreadGetProcessExitCode: no such process 16891
Game process updated : AppID 1284410 "/home/xan/.steam/debian-installation/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/home/xan/.steam/debian-installation/steamapps/common/SteamLinuxRuntime_soldier'/_v2-entry-point --verb=waitforexitandrun -- '/home/xan/.steam/debian-installation/steamapps/common/Proton - Experimental'/proton waitforexitandrun  '/home/xan/.steam/debian-installation/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'", ProcID 17058, IP 0.0.0.0:0
Setting breakpad minidump AppID = 1284410
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561198074477531 [API loaded no]
ThreadGetProcessExitCode: no such process 17062
ThreadGetProcessExitCode: no such process 17137
ThreadGetProcessExitCode: no such process 17117
ThreadGetProcessExitCode: no such process 17103
ThreadGetProcessExitCode: no such process 17097
ThreadGetProcessExitCode: no such process 17091
ThreadGetProcessExitCode: no such process 17067
ThreadGetProcessExitCode: no such process 17064
ThreadGetProcessExitCode: no such process 17054
pid 17060 != 17059, skipping destruction (fork without exec?)
Game process removed: AppID 1284410 "/home/xan/.steam/debian-installation/ubuntu12_32/reaper SteamLaunch AppId=1284410 -- '/home/xan/.steam/debian-installation/steamapps/common/SteamLinuxRuntime_soldier'/_v2-entry-point --verb=waitforexitandrun -- '/home/xan/.steam/debian-installation/steamapps/common/Proton - Experimental'/proton waitforexitandrun  '/home/xan/.steam/debian-installation/steamapps/common/GWENT The Witcher Card Game/REDprelauncher.exe'", ProcID 17058 
ThreadGetProcessExitCode: no such process 17076
ThreadGetProcessExitCode: no such process 17058
ThreadGetProcessExitCode: no such process 16890
ThreadGetProcessExitCode: no such process 16889
Game 1284410 created interface STEAMAPPLIST_INTERFACE_VERSION001 / AppList
Game 1284410 created interface STEAMAPPS_INTERFACE_VERSION008 / Apps
Game 1284410 created interface STEAMHTMLSURFACE_INTERFACE_VERSION_004 / HTMLSurface
Game 1284410 created interface STEAMHTTP_INTERFACE_VERSION002 / HTTP
Game 1284410 created interface STEAMINVENTORY_INTERFACE_V002 / Inventory
Game 1284410 created interface STEAMMUSICREMOTE_INTERFACE_VERSION001 / MusicRemote
Game 1284410 created interface STEAMMUSIC_INTERFACE_VERSION001 / Music
Game 1284410 created interface STEAMPARENTALSETTINGS_INTERFACE_VERSION001 / ParentalSettings
Game 1284410 created interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 / RemoteStorage
Game 1284410 created interface STEAMSCREENSHOTS_INTERFACE_VERSION003 / Screenshots
Game 1284410 created interface STEAMUGC_INTERFACE_VERSION010 / UGC
Game 1284410 created interface STEAMUSERSTATS_INTERFACE_VERSION011 / UserStats
Game 1284410 created interface STEAMVIDEO_INTERFACE_V002 / Video
Game 1284410 created interface SteamController006 / Controller
Game 1284410 created interface SteamFriends015 / Friends
Game 1284410 created interface SteamMatchMaking009 / Matchmaking
Game 1284410 created interface SteamMatchMakingServers002 / MatchmakingServers
Game 1284410 created interface SteamNetworking005 / Networking
Game 1284410 created interface SteamUser019 / User
Game 1284410 created interface SteamUtils009 / Utils
Game 1284410 method call count for IClientAppManager::GetAppInstallState : 2
Game 1284410 method call count for IClientUtils::RecordSteamInterfaceCreation : 22
Game 1284410 method call count for IClientUtils::GetSteamUILanguage : 1
Game 1284410 method call count for IClientUtils::GetAppID : 24
Game 1284410 method call count for IClientUser::GetSteamID : 1
Uploaded AppInterfaceStats to Steam
```

</details>

**Comment by monad0** [comment]
I also cannot launch the game since it updated and found mention of the same error related to mfc140u.dll. What would be the recommended way to install this DLL?
E: For now I just manually copied it from another game's prefix.

**Comment by terzag** [comment]
It might be worth mentioning a possible workaround: according to someone on Reddit, if you own Thronebreaker on Steam, you can launch Gwent from its menu and it seems to work properly. I haven't been able to confirm that given that I don't have TB on Steam.

**Comment by miseriae** [comment]
Had the same problem after update. Found that solution on reddit, I hope it can help you guys. 
https://www.reddit.com/r/gwent/comments/umizhv/comment/i8ag7yg/?utm_source=share&utm_medium=web2x&context=3

**Comment by terzag** [comment]
I've tried a different method that someone else mentioned in the Reddit thread, which is to copy mfc140u.dll in Gwent's prefix and it fixed the issue for me. It's probably better than replacing the REDprelauncher.exe (which can be reverted at any time with an update or verifying files).

You'll have to find a game installed that has this dll or find it elsewhere if you don't have one (not sure if the exact version matters). It's in [steamapps]/compatdata/[game ID]/pfx/drive_c/windows/system32 and syswow64. Copy it in [steamapps]/compatdata/1284410/pfx/drive_c/windows/system32 and syswow64.

If you can't find one in your installed games, it can probably be installed through winetricks/protontricks with a Visual C++ or .NET redistributable but I'm not sure which one would work.

It's a bit weird that this fixed it for me, though, as there's no mention of this dll in my log and people with a similar issue had it before the 10.5 update that broke the game.

**Comment by kisak-valve** [comment]
CANT DUEL WITH MY FRIEND USING STEAM LAUNCHER GWENT

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6873.
@gendonholaholo posted on 2023-06-24T15:09:27:

# Compatibility Report
- Name of the game with compatibility issues: GWENT
- Steam AppID of the game: 1284410

## System Information
- GPU: RTX 4090
- Driver/LLVM version: 525.105.17
- Kernel version: 6.1.35-x64v3-xanmod1
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: GE-Proton7-55

## I confirm:
- [ yes] that I haven't found an existing compatibility report for this game.
- [ yes] that I have checked whether there are updates for my system available.



## Symptoms 
CANT DUEL WITH MY FRIEND USING STEAM LAUNCHER GWENT

## Reproduction
